aries_cloudagent/protocols/coordinate_mediation/v1_0/handlers/mediation_grant_handler.py

```diff
@@ -54,7 +55,7 @@ async def handle(self, context: RequestContext, responder: BaseResponder):
                     )
 
             # Set to default if metadata set on connection to do so
-            async with context.session() as session:
+            async with profile.session() as session:
```

Comment:
Here it also works if I removed it, inside the function it was just called a storage inject
should I remove it?